Sources/Observer.swift

```diff
@@ -18,6 +18,24 @@ extension Signal {
 		/// Whether the observer should send an `interrupted` event as it deinitializes.
 		private let interruptsOnDeinit: Bool
 
+		/// The target observer of `self`.
+		private let wrapped: AnyObject?
```

Comment:
Sadly, we have to retain stuff marked with `interruptsOnDeinit`, and retaining it using a thunk adds an overhead to all calls to the observer, as opposed to an 8 byte storage of a reference. :/

```swift
// @testEventTransformingCoreMapFilter(): avg 6091 ns; min 5250 ns
self.action = transform(observer.action)
self.wrapped = observer

// @testEventTransformingCoreMapFilter(): avg 6406 ns; min 5428 ns
self.action = transform { observer.action($0) }
```